src/test/java/org/terasology/launcher/TestingUtils.java

```diff
@@ -109,4 +76,80 @@ public boolean hasEntry(String job, int build) {
         }
     }
 
+    public static class MockBuildRepository implements IBuildRepository {
```

Comment:
Why are you using an inner class here? Should we move it to its own file?

Reply:
I don't have much of an opinion on that, my sense of Java conventions for this kind of thing is not very well-developed. It is a test helper, and it works very closely with VersionInformation.

Easy enough to move if that would be more consistent with the coding standards.

Reply:
or we could move it to DELETE